### PR TITLE
Remove link from Device Plugins page to KEP

### PR DESCRIPTION
**Pull request by omBratteng**
Update the link to the proposal on the [Device Plugins](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/) page. Previously pointed to archived repository, now points to enhancements repo and the specific KEP

### CODE REVIEW

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ weight: 20
 <!-- overview -->
 {{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
-Kubernetes provides a [device plugin framework](https://git.k8s.io/design-proposals-archive/resource-management/device-plugin.md)
+Kubernetes provides a [device plugin framework](https://git.k8s.io/enhancements/keps/sig-node/3573-device-plugin/README.md)
```
Comment (sftim) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:13]: We shouldn't link to https://git.k8s.io/enhancements/keps/sig-node/3573-device-plugin/README.md with this link text, as that page doesn't mention the word _framework_ anywhere. This page itself is about device plugins.

I would omit the link here. We can link to https://kep.k8s.io/3573 in the **What's next** section at the foot of the page, if we want to.

**Comment by bradtopol** [comment]
Assigning @sftim since he is already engaged on this.
/assign @sftim

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md (hunk 1 above)
Comment (omBratteng) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:13]: Hmm, yeah, makes sense.
Not sure what text would suit the link in the **What's next** section

Comment (sftim) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:13]: How about removing the hyperlink?

**Comment by sftim** [comment]
/retitle point Device Plugins page to KEP

**Comment by sftim** [comment]
/retitle Remove link from Device Plugins page to KEP

@omBratteng please rebase this against _main_ (see https://lwkd.info/2023/20230606#developer-news for context). If you're not sure how to, please reply here to say so and we'll try to help.

**Comment by sftim** [comment]
If you're willing to rebase rather than merge in _main_, we'd prefer that.

**Comment by omBratteng** [comment]
I did a rebase after I saw that I first had merged main

**Comment by omBratteng** [comment]
Just rebased to get the last commit on main as well

**Review by sftim — COMMENTED** [review]
/lgtm
/label tide/merge-method-squash

**Pull request merged**